roles/nfs/precheck/tasks/storage.yml

```diff
@@ -0,0 +1,24 @@
+
+- block:
+
+  - name: nfs | Create extra facts
+
+    set_fact:
+
+     nfs_params: "{{ nfs | default([]) | combine(item)}}"
+
+    when: 
+
+    - nfs is defined
+
+    with_items:
+
+    - "{{ nfs }}"
+
+      # - debug:
+
+      # msg: "{{ nfs_params }}"
+
```

Comment:
@shubh1410  We are not using storage.yml file , so please remove this file from commit.

Reply:
@rajan-mis suggested changes implemented